metafacture-io/src/main/java/org/metafacture/io/HttpOpener.java

```diff
@@ -51,18 +54,55 @@ public final class HttpOpener extends DefaultObjectPipe<String, ObjectReceiver<R
 
     private static final String ACCEPT_HEADER = "accept";
     private static final String ENCODING_HEADER = "accept-charset";
-
     private static final String ACCEPT_DEFAULT = "*/*";
+    private static final String CONTENT_TYPE_HEADER = "content-Type";
+    private static final String CONTENT_TYPE_DEFAULT = "application/json";
```

Comment:
I'm not sure we should set JSON as default. Seems a little too opinionated.

Reply:
Used '*/*' as default (unsure if this is even a valid value).

Reply:
No, it's not valid for content-type; the most appropriate default would [probably](https://datatracker.ietf.org/doc/html/rfc7231#section-3.1.1.5) be `application/octet-stream`.

Reply:
Or just don't set a default at all.